src/ripple/server/impl/Port.cpp

```diff
@@ -217,6 +217,21 @@ parse_Port (ParsedPort& port, Section const& section, std::ostream& log)
     set(port.ssl_cert, "ssl_cert", section);
     set(port.ssl_chain, "ssl_chain", section);
     set(port.ssl_ciphers, "ssl_ciphers", section);
+
+    port.pmd_options.server_enable =
+        section.value_or("permessage_deflate", false);
+    port.pmd_options.client_max_window_bits =
+        section.value_or("client_max_window_bits", 15);
+    port.pmd_options.server_max_window_bits =
+        section.value_or("server_max_window_bits", 15);
+    port.pmd_options.client_no_context_takeover =
+        section.value_or("client_no_context_takeover", false);
+    port.pmd_options.server_no_context_takeover =
+        section.value_or("server_no_context_takeover", false);
+    port.pmd_options.compLevel =
+        section.value_or("compress_level", 3);
+    port.pmd_options.memLevel =
+        section.value_or("memory_level", 4);
```

Comment:
1. Nope, the Server never operates in client mode. And we don't want compression for the client we use for testing (that would just slow the tests down with no benefit)
2. For a client, a default of 8 is okay but on a server that would crush the CPU, imagine 1,000 connections all trying to go for max compression. I made it 3 to be less resource intensive by default. Of course, if you are into pain you can always dial it up...